### PR TITLE
Lesson_01_basics

### PR DESCRIPTION
**Pull request by synulewar**
@kuszcjan

### CODE REVIEW

File: students/synowiec_krzysztof/lesson_01_basics/clockFaceOne.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+def main():
+    anglePerSecond = (3600 * 12)/360
```
Comment (kuszcjan) [inline students/synowiec_krzysztof/lesson_01_basics/clockFaceOne.py]: it's okay, but sometimes it's good to write like:
`anglePerSecond` = (60 * 60 * 12)/360`
to make thing easier to understand and show how some values are build

File: students/synowiec_krzysztof/lesson_01_basics/helloHarry.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+def main():
+    name = input()
+    print("Hello, " + name)
+
+if __name__ == '__main__':
+    main()
```
Comment (kuszcjan) [inline students/synowiec_krzysztof/lesson_01_basics/helloHarry.py]: Exclamation mark at the end is missing!

Please check code at snakify if it comply with all requirements and if it passes all tests:
https://snakify.org/lessons/print_input_numbers/problems/hello_harry/

File: students/synowiec_krzysztof/lesson_01_basics/carRoute.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import math
+
+def main():
+    distancePerDay = int(input())
+    distnce = int(input())
```
Comment (kuszcjan) [inline students/synowiec_krzysztof/lesson_01_basics/carRoute.py]: just a typo : distnce  -> distance

File: students/synowiec_krzysztof/lesson_01_basics/threeDigitNumber.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (kuszcjan) [inline students/synowiec_krzysztof/lesson_01_basics/threeDigitNumber.py]: Please name files to reflect snakify problem, eg sum_of_digits.py

**Comment by kuszcjan** [comment]
there is different naming convetions for python files:

"So PEP 0008 tells you that:
modules (filenames) should have short, all-lowercase names, and they can contain underscores;"
e.g.: appleShare.py -> apple_share.py
https://softwareengineering.stackexchange.com/questions/308972/python-file-naming-convention

Minor (convetion) thing: There should be newline at the end of file.

**Comment by synulewar** [comment]
Done

**Comment by kuszcjan** [comment]
Nice.

**Pull request merged**